docs/src/reference/index.md

```diff
@@ -50,6 +50,7 @@ Check the index on the left for a more detailed description of any symbol.
 | [`EventSet.enumerate()`][temporian.EventSet.enumerate]                                                     | Creates an ordinal feature enumerating the events according to their timestamp.                              |
 | [`EventSet.experimental_fast_fourier_transform()`][temporian.EventSet.experimental_fast_fourier_transform] | Apply a Fast Fourier Transform.                                                                              |
 | [`EventSet.filter()`][temporian.EventSet.filter]                                                           | Filters out events in an [`EventSet`][temporian.EventSet] for which a condition is false.                    |
+| [`EventSet.filter_max_moving_count()`][temporian.EventSet.filter_max_moving_count]                         | Skips events such that no more than `max_count` events are within a time window of `window_length`           |
```

Comment:
missing dot at the end

Reply:
also update with docstring without `max_count`

Reply:
*Done x2